.ci/install_kata_image_nosystemd.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+cidir=$(dirname "$0")
+rust_agent_repo="github.com/kata-containers/kata-containers"
+osbuilder_repo="github.com/kata-containers/osbuilder"
+arch=$("${cidir}"/kata-arch.sh -d)
+
+build_image() {
+	go get -d "${osbuilder_repo}" || true
+	pushd "${GOPATH}/src/${osbuilder_repo}/rootfs-builder"
+	export ROOTFS_DIR="${GOPATH}/src/${osbuilder_repo}/rootfs-builder/rootfs"
+
+	#Currently, only alpine support no systemd"
+	distro="alpine"
+	sudo -E GOPATH="${GOPATH}" USE_DOCKER=true SECCOMP=no ./rootfs.sh "${distro}"
+	install -o root -g root -m 0550 -T ../../agent/kata-agent ${ROOTFS_DIR}/sbin/init
```

Comment:
not sure but I think this won't work, alpine is built around musl libc, did `./rootfs.sh` build this binary using musl?

Reply:
sorry to have missed this comments so long.
I have tried this script both on x86 and arm64 and it works well. when I inspect the kata-agent binary using file, it shows:
kata-agent: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib/ld-musl-x86_64.so.1, not stripped
Does that mean this kata-agent build using musl?

Reply:
thanks for confirming, I  guess I was wrong